src/typst.ts

```diff
@@ -0,0 +1,376 @@
+      options: [
+        {
+          name: "--font-path",
+          description: "Adds additional directories to search for fonts",
+          isRepeatable: true,
+          args: {
+            name: "font_paths",
+            template: "filepaths",
+          },
+        },
+        {
+          name: "--variants",
+          description: "Also lists style variants of each font family",
+        },
+        {
+          name: ["-h", "--help"],
+          description: "Print help",
+        },
+      ],
+    },
+    {
+      name: "update",
+      description: "Self update the Typst CLI (disabled)",
+      icon: "🆙",
+      options: [
+        {
+          name: "--force",
+          description:
+            "Forces a downgrade to an older version (required for downgrading)",
+          exclusiveOn: ["--revert"],
+        },
+        {
+          // TODO: Make it impossible to use with one or more of the other specified arguments (e.g version)
+          name: "--revert",
+          description:
+            "Reverts to the version from before the last update (only possible if `typst update` has previously ran)",
+          exclusiveOn: ["--force"],
+        },
+        {
+          name: ["-h", "--help"],
+          description: "Print help",
+        },
+      ],
+      args: {
+        name: "version",
+        description: "Which version to update to (defaults to latest)",
+        isOptional: true,
+      },
+    },
+    {
+      name: "help",
+      description: "Print this message or the help of the given subcommand(s)",
+      icon: "ℹ️",
+      subcommands: [
+        {
+          name: "compile",
+          description: "Compiles an input file into a supported output format",
+          icon: "📄",
+        },
+        {
+          name: "watch",
+          description: "Watches an input file and recompiles on changes",
+          icon: "🔄",
+        },
+        {
+          name: "query",
+          description: "Processes an input file to extract provided metadata",
+          icon: "🔍",
+        },
+        {
+          name: "fonts",
+          description:
+            "Lists all discovered fonts in system and custom font paths",
+          icon: "🔤",
+        },
+        {
+          name: "update",
+          description: "Self update the Typst CLI (disabled)",
+          icon: "🆙",
+        },
+        {
+          name: "help",
+          description:
+            "Print this message or the help of the given subcommand(s)",
+          icon: "ℹ️",
+        },
+      ],
+    },
+  ],
+  options: [
+    {
+      name: "--cert",
+      description:
+        "Path to a custom CA certificate to use when making network requests",
+      args: {
+        name: "cert",
+        template: "filepaths",
+      },
+    },
+    {
+      name: ["-v", "--verbosity"],
+      description:
+        "Sets the level of logging verbosity: -v = warning & error, -vv = info, -vvv = debug, -vvvv = trace",
+      isRepeatable: 4,
+    },
```

Comment:
A bit of an edge case, but I think it makes sense to split these into two options.

```suggestion
    {
      name: ["-v"],
      description:
        "Sets the level of logging verbosity",
      isRepeatable: 4,
    },
    {
         name: ["--verbosity"],
         args: {
            name: "level",
            suggestions: ["warning", "error", "info", "debug", "trace"]
         }
    }
    
```

Reply:
Right. An option like that `--verbosity` might be easier to use. Since I am not an [typst](https://github.com/typst/typst) maintainer, it might be good if you could suggest it for typst.

Reply:
I believe this is verbatim how the command works right now, you can literally repeat `--verbosity` up to 4 times.